src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball firstBall = lottery.getRandomBall();
+        Ball secondBall = lottery.getRandomBall();
+        Ball thirdBall = lottery.getRandomBall();
+        Ball[] balls = new Ball[]{firstBall,secondBall,thirdBall};
+        for (Ball ball : balls) {
+            System.out.println(ball.toString());
```

Comment:
we don't need to explicitly call .toString() on objects which we pass to system out println, it happens under the hood